scripts/hero1-phone-ui.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
Please add this also to the install_systemd_autostart_hero1

Reply:
Done. Please test on the robot whether it works correctly.